install-civs

```diff
@@ -14,9 +15,8 @@ then
   exit 1;
 fi
 
-. $1 || exit 1
-
 echo "Read settings file."
+. $1 || exit 1
```

Comment:
This was meant to be a past tense of "read'.

Reply:
Ah! Heh, yes. :)